README.md

```diff
@@ -59,8 +64,16 @@ func main() {
 
 	c.SubscribeTopics([]string{"myTopic", "^aRegex.*[Tt]opic"}, nil)
 
-	for {
-		msg, err := c.ReadMessage(-1)
+	sig := make(chan os.Signal, 1)
+	signal.Notify(sig, syscall.SIGINT)
```

Comment:
Please change this to os.Interrupt instead of syscall.SIGINT, and remove the import for syscall, to ensure this code works across platforms including Windows. 

Here's more info for signals - https://pkg.go.dev/os#Signal - the part where it mentions that the we are guaranteed only 2 signal values across platforms (os.Interrupt and os.Kill)

Minor - if you can, add a comment above the `sig := ...` line explaining the purpose of this goroutine.